src/common/components/signUpModal/signUpModal.tpl.html

```diff
@@ -74,12 +75,12 @@
              type="password"
              class="form-control form-control-subtle"
              ng-model="$ctrl.password"
-             minlength="8"
+             minlength="15"
```

Comment:
This is not a universal rule.

Reply:
Gotcha, so the universal rule for passwords is that it has to be greater than 8 characters, but we've added an extra layer of security to make it at least 15.

As I continue to edit the give-web site, is there a list of universal rules? So I know what I can and can't change.

Reply:
These rules are defined within Okta itself, they aren't give site rules. I'm not sure how to find that logic to be honest.